### PR TITLE
Use 127.0.0.1 instead of localhost to point to an IPV4 adress

### PR DESCRIPTION
**Pull request by Philipou**
Everything is in the title :)

### CODE REVIEW

**Comment by kolbytn** [comment]
Was this necessary because of your os?

**Comment by Philipou** [comment]
Oh yes probably I did not test in linux but It may work but I think if it's set to 127.0.0.1 it will work in both i think

**Comment by kolbytn** [comment]
Can you resolve the merge conflicts in local.js please.

**Comment by Philipou** [comment]
Should be ok now sorry if it's not i don't do this often :)

**Review by MaxRobinsonTheGreat — APPROVED** [review]
looks good. @kolbytn good to merge?

File: src/models/local.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export class Local {
     constructor(model_name, url) {
         this.model_name = model_name;
-        this.url = url || 'http://localhost:11434';
+        this.url = 'http://127.0.0.1:11434';
```
Comment (kolbytn) [inline src/models/local.js:4]: This needs to be `url || 'http://127.0.0.1:11434'`

**Review by kolbytn — CHANGES_REQUESTED** [review]

**Comment by Philipou** [comment]
Fixed

**Comment by MaxRobinsonTheGreat** [comment]
good to merge @kolbytn?

**Comment by kolbytn** [comment]
It doesn't look like @Philipou pushed his fix.

**Comment by Philipou** [comment]
hmm that's weird i did fix it but i was in the bus i did it directly on my phone maybe that's the reason... welp sorry about all this mess but we have #73 now thanks ! Closing now

**Pull request closed without merging**